samples/reactjs-todo/project.json

```diff
@@ -7,6 +7,7 @@
       "executor": "@nrwl/web:webpack",
       "outputs": ["{options.outputPath}"],
       "options": {
+        "compiler": "swc",
```

Comment:
again - faster than babel, but does not type check, type check is still done by tsc.